### PR TITLE
Feature/ra 71 build secrets console

### PR DESCRIPTION
**Pull request by ingeknudsen**

### CODE REVIEW

File: src/components/page-configuration/build-secrets-toggler.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import useGetBuildSecrets from '../page-build-secret/use-get-build-secrets';
+
+import * as routing from '../../utils/routing';
+
+const BuildSecretsToggler = props => {
+  const getBuildSecretsState = useGetBuildSecrets(props.appName);
+  const data = getBuildSecretsState.data;
+
+  return (
+    <Panel>
```
Comment (keaaa) [inline src/components/page-configuration/build-secrets-toggler.js]: not sure if its worth it, but maybe you can generalise `build-secret-toggler.js` and `image-hubs-toggler.js` and use a common component

File: src/components/page-build-secret/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.build-secrets-overview {
```
[the remark this answers is not in the document]
Comment (ingeknudsen) [reply src/components/page-build-secret/style.css]: Done

File: src/components/page-build-secret/overview.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const overview = props => (
+  <div className="build-secrets-overview">
```
[the remark this answers is not in the document]
Comment (ingeknudsen) [reply src/components/page-build-secret/overview.js]: Done

File: src/components/page-application/index.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ import PageDeployments from '../page-deployments';
 import PageDeployment from '../page-deployment';
 import PageEnvironment from '../page-environment';
 import PageEnvironments from '../page-environments';
+import PageBuildSecrets from '../page-build-secret';
```
[the remark this answers is not in the document]
Comment (ingeknudsen) [reply src/components/page-application/index.js]: Done

**Pull request merged**